xml/System.Reflection.Emit/OpCodes.xml

```diff
@@ -10065,7 +10065,7 @@ The following <xref:System.Reflection.Emit.ILGenerator.Emit%2A> method overloads
   
 1.  The address stored in the local variable at the specified index is pushed onto the stack.  
   
- The `ldloca` instruction pushes the address of the local variable number at the passed index onto the stack, where local variables are numbered 0 onwards. The value pushed on the stack is already aligned correctly for use with instructions like <xref:System.Reflection.Emit.OpCodes.Ldind_I> and <xref:System.Reflection.Emit.OpCodes.Stind_I>. The result is a transient pointer (type `*`).  
+ The `ldloca` instruction pushes the address of the local variable number at the passed index onto the stack, where local variables are numbered 0 onwards. The value pushed on the stack is already aligned correctly for use with instructions like <xref:System.Reflection.Emit.OpCodes.Ldind_I> and <xref:System.Reflection.Emit.OpCodes.Stind_I>. The result is a managed pointer (type `&`) which can however be safely converted to a transient pointer (type `*`) with <xref:System.Reflection.Emit.OpCodes.Conv_U>.
```

Comment:
```suggestion
 The `ldloca` instruction pushes the address of the local variable number at the passed index onto the stack, where local variables are numbered 0 onwards. The value pushed on the stack is already aligned correctly for use with instructions like <xref:System.Reflection.Emit.OpCodes.Ldind_I> and <xref:System.Reflection.Emit.OpCodes.Stind_I>. The result is a managed pointer (type `&amp;`). The local variable is stored in unmanaged memory, so the return value can be converted to an unmanaged pointer without pinning.
```

Reply:
I think `conv.u` should still be mentioned there.

Reply:
I disagree. I would prefer to keep this as close to ECMA-335 modified with the relavantly applied addendums.